common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/ApacheBlockingContainerClient.scala

```diff
@@ -149,10 +152,9 @@ protected class ApacheBlockingContainerClient(hostname: String,
     } match {
       case Success(response) => response
       case Failure(t: RetryableConnectionError) if retry =>
-        val sleepTime = 50.milliseconds
         if (timeout > Duration.Zero) {
-          Thread.sleep(sleepTime.toMillis)
-          val newTimeout = timeout - sleepTime
+          Thread.sleep(50)
```

Comment:
I'm fine with this change. At the same time, I'm a strong supporter of using units whenever possible for measurable quantities. What about following proposal to make it explicit:

```
Thread.sleep(50) // milliseconds
```

With this comment, it's clear that you want to wait for 50 ms.

See here for an example where missing / wrong units caused the loss of a space probe... :-)

Reply:
@sven-lange-last, added a comment denoting the units used for `Thread.sleep()`.